.github/workflows/basic_cli_build_release.yml

```diff
@@ -240,6 +240,8 @@ jobs:
           mkdir platform
           # move all files to platform dir
           find . -maxdepth 1 -type f -exec mv {} platform/ \;
+          # move all dirs (except roc_nightly) to platform dir
+          find . -maxdepth 1 -type d ! -name 'platform' ! -name 'roc_nightly' -exec mv {} platform/ \;
```

Comment:
Arg is now a directory instead of a file (Arg.roc) so it needs to be moved as well